### ISSUE TITLE
Add proper licenses to project

### ISSUE THREAD

**Issue by ScriptedAlchemy** [web-infra-dev/bundler-benchmark#3]
Since this repo is heavily based off webpack/benchmark - we should add the matching licenses similar to webpacks original benchmark project has.

**Comment by IWANABETHATGUY** [comment]
Yeah, we would like to add the matching licenses similar to webpacks original benchmark project, but it seems that 
![image](https://user-images.githubusercontent.com/17974631/224992298-449fe545-9909-4adf-8995-a6e908860c75.png)
https://github.com/webpack/benchmark don't have a license for now, so would you mind adding a proper license? Or, I could  open a pull request to add a license If the webpack team would like to accept it..

**Comment by IWANABETHATGUY** [comment]
@ScriptedAlchemy Any idea? sorry for the ping.

**Comment by ScriptedAlchemy** [comment]
Hey! Gotcha, let me take a stab tomorrow and see if I can source license attribution, otherwise maybe we use rspack mit license and perhaps add one extra line for attribution
